### DISCUSSION TITLE
workflow for the analysis of data acquired from GCTOFMS Pegasus BT, Leco

### DISCUSSION THREAD

**Discussion by kilsunlee** [systemsomicslab/MsdialWorkbench#384] [Q&A]
Does anyone has workflow for the analysis of data acquired from GCTOFMS Pegasus BT, Leco?  When I use files converted to .cdf, MS-DIAL crashes  (ver. 4.9 or ver. 5.1).  When I use mzMl files, it shows error message in reading deconvolution files.

**Comment by aicha1234444** [comment]
I have the same problem. Could you please share if you found a solution?

**Reply by kilsunlee** [reply]
I could finish the analysis installing Microsoft Visual J# Version 2.0 Redistributable Package, which enabled the conversion of .cdf to .abf.
However, after conclusion of the analysis, I could not export the aligned results. Thus I have for while gave up. Please, if you have further solution, let me know. 
Thank you
